zpay32/fuzz_test.go

```diff
@@ -12,15 +12,7 @@ import (
 
 func FuzzDecode(f *testing.F) {
 	f.Fuzz(func(t *testing.T, data string) {
-		inv, err := Decode(data, &chaincfg.TestNet3Params)
-		if err != nil {
-			return
-		}
-
-		// Call these functions as a sanity check to make sure the
-		// invoice is well-formed.
-		_ = inv.MinFinalCLTVExpiry()
-		_ = inv.Expiry()
+		_, _ = Decode(data, &chaincfg.TestNet3Params)
```

Comment:
I think it would be a good idea to get the fuzzer to randomize the chaincfg

Reply:
Done in 7390e6d23b6fd8329956a278892451ea6f0edb5f.